src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -108,12 +108,13 @@ public async Task<IList<string>> AutocompletePackageVersionsAsync(string id, boo
             return await _httpClient.GetJsonAsync<List<string>>(uri, logger);
         }
 
-        public async Task PushAsync(Stream nupkgStream, ITestOutputHelper logger, bool isSymbolsPackage = false)
+        public async Task PushAsync(Stream nupkgStream, ITestOutputHelper logger, PackageType packageType)
         {
             var galleryEndpoint = await GetGalleryUrlAsync(logger);
-            var url = isSymbolsPackage
-                ? $"{galleryEndpoint}/api/v2/symbolpackage"
-                : $"{galleryEndpoint}/api/v2/package";
+
+            var url = packageType != PackageType.SymbolsPackage
+                ? $"{galleryEndpoint}/api/v2/package"
+                : $"{galleryEndpoint}/api/v2/symbolpackage";
```

Comment:
change this to switch case with appropriate values set for the intended package types.

Reply:
I see there are a lot of different package types, I would just default to `/api/v2/package` and case for `symbolPackage` to the other endpoint.

Reply:
Move to switch now.